### PR TITLE
Add a test for mysql write and read flows

### LINKED ISSUE #77: Add a test for using mysql
A test should be added for the usage of reading / writing to mysql database.
The test can be run without Fybrik.

### PR DESCRIPTION
**Pull request by revit13**
/Closes #77 

This PR adds a test for reading and writing to mysql with fybrik,

### CODE REVIEW

File: hack/test-script/asset.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+apiVersion: katalog.fybrik.io/v1alpha1
+kind: Asset
+metadata:
+  name: userdata
+spec:
+  secretRef: 
+    name: userdata
+  details:
+    dataFormat: csv
```
Comment (cdoron) [inline hack/test-script/asset.yaml:9]: does it work without specifying `csv`?

File: hack/test_module.sh [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+# apply modules
+
+# Related to https://github.com/cert-manager/cert-manager/issues/2908
+# Fybrik webhook not really ready after "helm install --wait"
+# A workaround is to loop until the module is applied as expected
+if [ $moduleVersion == "master" ]
+then
+	CMD="${TOOLBIN}/kubectl apply -f ../module.yaml -n fybrik-system"
+else
+	CMD="${TOOLBIN}/kubectl apply -f https://github.com/fybrik/arrow-flight-module/releases/download/v$moduleVersion/module.yaml -n fybrik-system"
```
Comment (cdoron) [inline hack/test_module.sh:98]: should this be the `airbyte-module` rather than the `arrow-flight-module`?

File: hack/test_module.sh [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+    sleep 1
+done
+
+kubectl wait --for=condition=ready --all pod -n fybrik-blueprints --timeout=120s
+# do the reading using airbyte module
+kubectl exec -it my-shell -n default -- python3 /root/client.py --host my-app-read-fybrik-airbyte-sample-airbyte-module.fybrik-blueprints --port 80 --asset fybrik-airbyte-sample/userdata > ${tmp_dir}/out_fybrik_read.txt
+
+# check that what was written using airbyte module is identical to what was read using airbyte module.
+# skip first column from what was read as its the index.
+awk '(NR>1)' ${tmp_dir}/out_fybrik_write.txt | awk {'print $1$2$3'} > ${tmp_dir}/fybrik_write.txt
```
Comment (cdoron) [inline hack/test_module.sh:214]: should there be a `comma`? something like:
```
'print $1,$2,$3'
```
?

File: hack/test_module.sh (hunk 2 above)
Comment (revit13) [reply hack/test_module.sh:98]: Fixed thanks

File: hack/test_module.sh (hunk 3 above)
Comment (revit13) [reply hack/test_module.sh:214]: it works without the comma...

File: hack/test-script/asset.yaml (hunk 1 above)
Comment (revit13) [reply hack/test-script/asset.yaml:9]: no, I think that fybrik looks for a match between the dataformat in the catalog asset connection and the one in the module which includes:
```
        - source:
            protocol: mysql
            dataformat: csv

File: hack/test_module.sh (hunk 3 above)
Comment (revit13) [reply hack/test_module.sh]: Fixed. Thanks

**Review by cdoron — APPROVED** [review]

**Pull request merged**